src/sentry/releasehealth/base.py

```diff
@@ -1,31 +1,36 @@
 from datetime import datetime
-from typing import Dict, Optional, Sequence
+from typing import Mapping, Optional, Sequence, Tuple
 
 from typing_extensions import TypedDict
 
 from sentry.utils.services import Service
 
+ProjectId = int
+OrganizationId = int
+ReleaseName = str
```

Comment:
I do not think these necessary tbh and I think they complicate things for functions that just return str or int because then a `# type: ignore` is required .. str and int are already primitive so probably no need to define types from them
Plus it is harder to read.. initially I did not know what type `ReleaseName` was and had to look for it to realise its a `str`.. would have been much easier to read `str`

Reply:
This is a normal thing to do in strongly-typed languages. If you have two `ints` that mean two different things, this is useful to document what you mean even if it does not help type safety. Eventually you can try converting it to [mypy newtypes](https://mypy.readthedocs.io/en/stable/more_types.html#newtypes)

Snuba does the same thing (eg `RetentionDays`)

Reply:
I don't disagree with that but if using these types will mean that later you need to add `# type: ignore`s then I doubt the usefulness of creating custom types for ints and str

Reply:
That's not what's happening. I need to use `type: ignore` for other reasons.